backend/tests/e2e/test_api_e2e.py

```
@@ -219,6 +219,20 @@ def llm_response_handler(request):
                     # Determine response based on interaction count (simple pattern)
                     total_interactions = len(all_llm_interactions)
 
+                    # Define realistic token usage for each interaction
+                    # These values are used to test token tracking at interaction, stage, and session levels
```

Ruff (0.12.2)

Line too long (108 > 88)

(E501)